src/App/Fossa/Config/ConfigFile.hs

```diff
@@ -90,18 +94,23 @@ resolveConfigFile base path = do
   loc <- resolveLocation base path
   case loc of
     DefaultConfigLocation dir -> do
-      let realpath = dir </> defaultConfigFileName
-      exists <- doesFileExist realpath
-      if not exists
-        then -- File missing and not requested
-          pure Nothing
-        else do
-          configFile <- readContentsYaml realpath
+      possibleConfigFilePaths <-
+        mapM
```

Comment:
`mapM = traverse`, prefer `traverse`.  This should probably become an `hlint` rule.

Reply:
I can add this in a follow-up PR.